client/client.go

```diff
@@ -123,24 +152,69 @@ func getProjects(logger hclog.Logger, filter string, options ...option.ClientOpt
 		return nil, err
 	}
 
-	call := service.Projects.List()
-	if filter != "" {
-		call.Filter(filter)
+	var (
+		projects []string
+		inactive int
+	)
+
+	for _, folder := range folders {
+		call := service.Projects.List()
+		if folder != "" {
+			call.Parent(folder)
+		}
+
+		for {
+			output, err := call.Do()
+			if err != nil {
+				return nil, err
+			}
+			for _, project := range output.Projects {
+				if project.State == "ACTIVE" {
+					projects = append(projects, project.ProjectId)
+				} else {
+					logger.Info("Project state is not active. Project will be ignored", "project_id", project.ProjectId, "project_state", project.State)
+					inactive++
+				}
+			}
```

Comment:
maybe move this after the call or combine with the loop that removes dupes

Reply:
I don't understand, maybe you parsed it wrong? `projects` is a slice of project ids, not project objects.